### ISSUE TITLE
components/rook: Can't enable monitoring

### ISSUE THREAD

**Issue by johananl** [kinvolk/lokomotive#1025]
When deploying the `rook` component without specifying `enable_monitoring` and then re-applying the component with `enable_monitoring` set to `true`, the following error occurs:

```
Applying component 'rook'...
FATA[0001] upgrading release failed: [unable to recognize "": no matches for kind "PrometheusRule" in version "monitoring.coreos.com/v1", unable to recognize "": no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1"]  args="[rook]" command="lokoctl component apply"
```

Lokomotive revision: 3008e5a1cd7cc478482a7c2b365e8a02ae25fc95

## Steps to reproduce

Deploy the `rook` component (I used a Packet cluster in case it matters):

```hcl
component "rook" {}
```

Enable monitoring:

```hcl
component "rook" {
  enable_monitoring = true
}
```

Run `lokoctl component apply rook`.

cc: @surajssd

**Comment by invidian** [comment]
@johananl did you also configure `prometheus-operator` component?

**Comment by johananl** [comment]
I was following https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/monitoring-with-prometheus-operator.md which pointed me to https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/rook-ceph-storage.md which told me to do the steps described above.

I now see the following:

>NOTE: Deploy the prometheus-operator component before. For more information follow this doc.

So we have a "cyclic dependency" in the docs then. I'll leave this tagged as a bug and change it to a docs-related issue.

**Comment by invidian** [comment]
@johananl if we fix docs, can we close this issue? We already have https://github.com/kinvolk/lokomotive/issues/750 about the dependencies.

**Comment by johananl** [comment]
Yes, I'm fine with either solving the technical issue or explaining the limitation (i.e. the users need *some* sort of a solution, even if it's manual for now).
